ConfCoreTests/DecodingTests.swift

```diff
@@ -323,7 +323,7 @@ class AdapterTests: XCTestCase {
     }
 
     func testConfigAdapter() {
-        let config = loadFixture(from: "config", type: RootConfig.self)
```

Comment:
It's probably worth adding an "eventHero" key to the fixture data & testing the decoding of the object, for how little effort it takes?